bazel/repository_locations.bzl

```diff
@@ -1051,48 +1051,31 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "V8",
         project_desc = "Google’s open source high-performance JavaScript and WebAssembly engine, written in C++",
         project_url = "https://v8.dev",
-        # NOTE: Update together with com_googlesource_chromium_base_trace_event_common and com_googlesource_chromium_zlib.
-        version = "10.4.132.18",
+        # NOTE: Update together with com_googlesource_chromium_base_trace_event_common.
+        version = "10.7.193.13",
         # Static snapshot created using https://storage.googleapis.com/envoyproxy-wee8/wee8-fetch-deps.sh.
-        sha256 = "154223731091f531a3cb1a95ab9c14225983891240220a9e232c729358badd2c",
+        sha256 = "2170df76ce5d7ecd7fb8d131370d210152f200273cba126f06d8b88fb53c9fbc",
         urls = ["https://storage.googleapis.com/envoyproxy-wee8/v8-{version}.tar.gz"],
         use_category = ["dataplane_ext"],
         extensions = ["envoy.wasm.runtime.v8"],
-        release_date = "2022-07-11",
+        release_date = "2022-12-09",
```

Comment:
Hmmm? That would be 2 months in the future

Reply:
wierd - how is that passing dep checks

Reply:
ah - i see its not a github dep

Reply:
Ah, sorry I thought I copy-pasted it right from here: https://dev.azure.com/cncf/envoy/_build/results?buildId=118312&view=logs&jobId=1c04237c-5f52-5959-5f83-5522b3556331&j=1c04237c-5f52-5959-5f83-5522b3556331&t=db3f3889-c922-566b-fa2e-4bc4948bc467.

Reply:
From:

```
curl -v https://storage.googleapis.com/envoyproxy-wee8/v8-10.7.193.13.tar.gz > /dev/null
```

```
last-modified: Wed, 12 Oct 2022 20:05:56 GMT
```